skimage/feature/tests/test_util.py

```diff
@@ -37,20 +31,21 @@ def test_prepare_grayscale_input_2D():
 
 def test_mask_border_keypoints():
     keypoints = np.array([[0, 0], [1, 1], [2, 2], [3, 3], [4, 4]])
-    assert_equal(_mask_border_keypoints((10, 10), keypoints, 0),
-                 [1, 1, 1, 1, 1])
-    assert_equal(_mask_border_keypoints((10, 10), keypoints, 2),
-                 [0, 0, 1, 1, 1])
-    assert_equal(_mask_border_keypoints((4, 4), keypoints, 2),
-                 [0, 0, 1, 0, 0])
-    assert_equal(_mask_border_keypoints((10, 10), keypoints, 5),
-                 [0, 0, 0, 0, 0])
-    assert_equal(_mask_border_keypoints((10, 10), keypoints, 4),
-                 [0, 0, 0, 0, 1])
-
-
-@testing.skipif(plt is None, reason="Matplotlib not installed")
+    np.testing.assert_equal(_mask_border_keypoints((10, 10), keypoints, 0),
+                            [1, 1, 1, 1, 1])
+    np.testing.assert_equal(_mask_border_keypoints((10, 10), keypoints, 2),
+                            [0, 0, 1, 1, 1])
+    np.testing.assert_equal(_mask_border_keypoints((4, 4), keypoints, 2),
+                            [0, 0, 1, 0, 0])
+    np.testing.assert_equal(_mask_border_keypoints((10, 10), keypoints, 5),
+                            [0, 0, 0, 0, 0])
+    np.testing.assert_equal(_mask_border_keypoints((10, 10), keypoints, 4),
+                            [0, 0, 0, 0, 1])
+
+
+@pytest.mark.skipif(not has_mpl, reason="Matplotlib not installed")
```

Comment:
```suggestion
@pytest.mark.skipif(not has_mpl, reason="Needs matplotlib")
```
copying from `skimage/draw/tests/test_draw.py`